runtime/src/openxla/runtime/nvgpu/cudnn_api.cpp

```diff
@@ -93,16 +94,30 @@ CuDNNOperationGraph::~CuDNNOperationGraph() {
   graph_.reset();
 }
 
-const cudnn_frontend::OperationGraph& CuDNNOperationGraph::graph() const {
-  return *graph_;
+cudnn_frontend::OperationGraph& CuDNNOperationGraph::graph() { return *graph_; }
+
+//===----------------------------------------------------------------------===//
+// CuDNNExecutable.
```

Comment:
I think you wanted to have those without the `.`?

Reply:
Yeah, adding a little bit more consistency with IREE style